examples/redcross/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+org = "wso2"
+name = "persist_redcross"
```

Comment:
Shall we give a different name? like `basic_entity`

Reply:
Fixed in [f7c186b](https://github.com/ballerina-platform/module-ballerina-persist/pull/5/commits/f7c186b06ba8d5a2f9183a613d63d2cec1f1bdf5)